scenario/context/CometContext.ts

```diff
@@ -43,6 +43,7 @@ export type MigrationData = {
   migration: Migration<any>;
   lastProposal?: number;
   preMigrationBlockNumber?: number;
+  skip?: boolean;
```

Comment:
maybe `skipVerify` to be more specific about what we're skipping?

Reply:
Done